http.go

```diff
@@ -53,9 +53,16 @@ func NewHTTPTimestamper(httpClient *http.Client, endpoint string) (Timestamper,
 	if httpClient == nil {
 		httpClient = &http.Client{Timeout: 5 * time.Second}
 	}
-	if _, err := url.Parse(endpoint); err != nil {
+	tsaURL, err := url.Parse(endpoint)
+	if err != nil {
 		return nil, err
 	}
+	if tsaURL.Scheme == "" {
```

Comment:
Should we validate http/https?

Reply:
I'm seeing both http and https public tsa servers. (most them are actually http, for example, `http://timestamp.digicert.com`) @priteshbandi

Reply:
Added check that requires the scheme to be either `http` or `https`.